### DISCUSSION TITLE
v0.1.119 Game changer with Ollama Embeddings

### DISCUSSION THREAD

**Discussion by ginisksam** [open-webui/open-webui#1582] [General]
Just would like to update the team.
Finally - I can enjoy chatting with my docs.  (will abandon the tools and others frameworks now)
OS: Linux Mint 21.2
GPU: 2nd hand AMD 6700XT
Embeddings: mxbai-embed-large
ChunkSize: 1024
Doc Tested: https://www.gutenberg.org/cache/epub/16378/pg16378-images.html => pdf
docker: just 
$ docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:main
Results: Amazing
Thanks once again to the team.  Very good job.  Please keep it up.  Much appreciated.

**Comment by arjunkrishna** [comment]
how do you do this? 

I gave the embedding model as below in the docker statement

 docker run -d -p 3000:8081 -e OLLAMA_BASE_URL=http://host.docker.internal:11434 -e PORT=8081 -e DEFAULT_MODELS=mistral:latest -e MODEL_FILTER_ENABLED=True -e MODEL_FILTER_LIST=mistral:latest,llava:latest -e RAG_EMBEDDING_MODEL=mxbai-embed-large-v1 -e USER_PERMISSIONS_CHAT_DELETION=True  -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

for me it gave this error.
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://huggingface.co/sentence-transformers/mxbai-embed-large-v1/resolve/main/config.json

**Reply by justinh-rahb** [reply]
The variable you're referencing, `RAG_EMBEDDING_MODEL` is used for the internal RAG embed model of the WebUI, not related to Ollama embedding at all. I don't believe that's how `DEFAULT_MODELS` is intended to be used either... where did you get this information/instructions?

**Reply by arjunkrishna** [reply]
oops... i was just going through the list of env variables in this doc https://docs.openwebui.com/getting-started/env-configuration and seeing what it does. :)

**Reply by arjunkrishna** [reply]
updated it to 

docker run -d -p 3000:8081 -e OLLAMA_BASE_URL=http://host.docker.internal:11434 -e PORT=8081  -e MODEL_FILTER_ENABLED=True -e MODEL_FILTER_LIST="mistral:latest;llava:latest" -e USER_PERMISSIONS_CHAT_DELETION=True -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

will check more on ollama embedding.

**Comment by arjunkrishna** [comment]
found it
![image](https://github.com/open-webui/open-webui/assets/5271912/eef80b22-22a7-4fe0-82b7-4f72289c2e4c)

**Reply by mbentley** [reply]
Unless I missed it, I don't see a way to set that by default with environment variables on startup - or has anyone found something I am missing on that?
